test/addons/zlib-binding/binding.cc

```diff
@@ -52,4 +52,4 @@ inline void Initialize(v8::Local<v8::Object> exports,
 
 }  // anonymous namespace
 
-NODE_MODULE_CONTEXT_AWARE(binding, Initialize)
+NODE_MODULE_CONTEXT_AWARE(NODE_GYP_MODULE_NAME, Initialize)
```

Comment:
Is this an internal only name change? Would this change anything in the ecosystem?